framework/wazuh/core/utils.py

```diff
@@ -580,17 +580,31 @@ def safe_move(source, target, ownership=(common.wazuh_uid(), common.wazuh_gid())
     This function is useful to move files even when target directory is in a different filesystem from the source.
     Write permissions are required on target directory.
 
-    :param source: full path to source file
-    :param target: full path to target file
-    :param ownership: tuple in the form (user, group) to be set up after the file is moved
-    :param time: tuple in the form (addition_timestamp, modified_timestamp)
-    :param permissions: string mask in octal notation. I.e.: '0o640'
+    Parameters
+    ----------
+    source : str
+        Full path to source file
+    target : str
+        Full path to target file
+    ownership : tuple
+        Tuple in the form (user, group) to be set up after the file is moved
+    time : tuple
+        Tuple in the form (addition_timestamp, modified_timestamp)
+    permissions : str
+        String mask in octal notation. I.e.: '0o640'
```

Comment:
Parameter descriptions should end with a period:

```suggestion
    source : str
        Full path to source file.
    target : str
        Full path to target file.
    ownership : tuple
        Tuple in the form (user, group) to be set up after the file is moved.
    time : tuple
        Tuple in the form (addition_timestamp, modified_timestamp).
    permissions : str
        String mask in octal notation. I.e.: '0o640'.
```